handout/features/README.md

```diff
@@ -3,7 +3,7 @@
 ![ES6/TypeScript](../images/es6-typescript.png "ES6 and TypeScript")
 
 The language we usually call "JavaScript" is formally known as "EcmaScript".
-The new version of JavaScript, "EcmaScript 6" or "ES6", offers a number of new features that extend the power of the language.
-ES6 is not widely supported in today's browsers, so it needs to be transpiled to ES5. 
-You can choose between several transpilers, but we'll be using TypeScript, 
+The new version of JavaScript, "EcmaScript 2015" or "ES6", offers a number of new features that extend the power of the language.
```

Comment:
Actually ECMAScript 2015 is the same as ES6, the committed made the decision to use years. We should follow the recommendation. Here is something wikipedia:

https://en.wikipedia.org/wiki/ECMAScript#6th_Edition_-_ECMAScript_2015

Changed Ecma to ECMA.